.travis.yml

```diff
@@ -27,5 +27,5 @@ script:
   # - gipsy/scheme
   - PDO_INTERPRETER=gipsy make -C docker test
   # - wawaka/wasm
-  - PDO_INTERPRETER=wawaka WASM_MODE=INTERP WASM_MEM_CONFIG=MEDIUM make -C docker test
-  - PDO_INTERPRETER=wawaka WASM_MODE=INTERP_OPT WASM_MEM_CONFIG=MEDIUM make -C docker test
+  - PDO_INTERPRETER=wawaka WASM_MEM_CONFIG=MEDIUM make -C docker test
+  - PDO_INTERPRETER=wawaka-opt WASM_MEM_CONFIG=MEDIUM make -C docker test
```

Comment:
Odd. I did get a temporary 500 error from github earlier, this might be related? Though, to me it still appears as if Travis is working on the build (i.e. i don't see the red X yet). I'll update the travis.yml, we could definitely use a speed-up.

Reply:
I just got the an error as well. Did give me a link though to https://www.githubstatus.com/ and that shows that there are some incidents ...
BTW: the travis failure i got by going to the details page.  Strangely now, though, travis disappared! (But could also be due to your .travis.yaml change?  It looked a bit different than i would have expected (i though you could just define the different environment variable settings in a prefixed env-section) but there are always different roads to Rome :-)

Reply:
Yeah, I'm not sure what's going on with github. I was getting a bunch of 500 errors. Not sure why Travis disappeared completely. Before it was having trouble parsing the updated .travis.yml file, but I have run yamllint on it now, so that shouldn't be the issue :)

Reply:
Looks like we gained about 10-12 min in our build. Not bad :)